src/main/java/util/TokenFromMSAL4J.java

```diff
@@ -0,0 +1,103 @@
+package util;
+
+import com.microsoft.aad.msal4j.*;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Set;
+
+public class TokenFromMSAL4J {
```

Comment:
Did you consider using a client creds flow and getting a JWT programmatically?  Do you need the JWT for anything other than signing API requests?

Using a password flow isn't recommended by MS - appreciate that this is a limited use case in that its all non prod.

Reply:
Mike, We're re-writing the pack before long anyway, so as a temporary measure I'd like to get this merged in.